### DISCUSSION TITLE
Can't use Budibase on MacOS

### DISCUSSION THREAD

**Discussion by cabyz-stale** [Budibase/budibase#1569] [Help] [answered]
Hey guys! probably the stupidest question here but, I was using Budibase locally on Windows.

Then, tried to use my Mac and for some reason, I can't deploy `$ npm i -g @budibase/cli`

I have NPM and NodeJS installed, I was able with the digital-ocean one. But I can't use it because I don't have the Budibase app on Mac, I also installed Docker but not working either

**Comment by shogunpurple** [comment, marked as answer]
Hi @cabyz - what error are you getting when you run `npm i -g @budibase/cli`? You also no longer need the budibase desktop app to use budibase. We are now totally web based.

**Reply by cabyz-stale** [reply]
@shogunpurple lightning-fast I really appreciate it!

Then in that case, what can I do after I deploy the DigitalOcean droplet? if I go to the IPv4 there's nothing there, do I log into the terminal with SSH and add the user?

**Reply by shogunpurple** [reply]
@cabyz the digitalocean droplet is currently outdated - we are waiting on digitalocean approving our image which should be over the next day or two. 

You could however SSH into your digitalocean droplet and follow the steps here to get your instance set up:

https://docs.budibase.com/getting-started

**Reply by cabyz-stale** [reply]
Sorry for all the back and forth @shogunpurple 

I deployed a docker droplet with the manual guide because the digitalocean one didn't accept `$ npm i -g @budibase/cli` even after trying to install npm

so after creating the docker droplet, what do I need to do? I see that [this guide](https://docs.budibase.com/self-hosting/hosting-methods/docker-compose) may be the solution but it's focused on local dev so I'm a little lost

I guess that another option would be to just wait a couple of days and use the new droplet

**Reply by shogunpurple** [reply]
If you bring up a brand new docker droplet, SSH into it, then the following commands should work. (Run them one by one):

```
wget https://github.com/Budibase/budibase/releases/download/v0.9.1/cli-linux
chmod +x cli-linux
./cli-linux hosting --init
./cli-linux hosting --start
```

**Reply by cabyz-stale** [reply]
Thanks so much, @shogunpurple you're the man!

looking to build something with this amazing product you guys have

<img width="697" alt="Screen Shot 2021-05-26 at 5 23 31 p m" src="https://user-images.githubusercontent.com/67131949/119739026-3a991b00-be47-11eb-8698-71a1ffc214dd.png">
